staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -116,6 +116,13 @@ type RollingUpdateStatefulSetStrategy struct {
 	// Default value is 0.
 	// +optional
 	Partition *int32 `json:"partition,omitempty" protobuf:"varint,1,opt,name=partition"`
+	// The maximum number of pods that can be unavailable during the update.
+	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
+	// Absolute number is calculated from percentage by rounding up. This can not be 0.
+	// Defaults to 1. This field is alpha-level and is only honored by servers that enable the
+	// MaxUnavailableStatefulSet feature.
```

Comment:
This comment should explain how this field interacts with the Partition field.

...The partition field comment is also not helpful and could be expanded if you know what it is supposed to mean.

Reply:
it does not . maxUnavailable only determines at rolling update time, how many can we make go down, and partition limits the ordinals that can go down to the range [ replicas: partition]

Reply:
i will fix these comments, partition one is completely useless, i just noticed :-)